src/Illuminate/Mail/Transport/MandrillTransport.php

```diff
@@ -63,14 +63,20 @@ public function stop()
 	 */
 	public function send(Swift_Mime_Message $message, &$failedRecipients = null)
 	{
-		return $this->client->post('https://mandrillapp.com/api/1.0/messages/send-raw.json', [
-			'form_params' => [
-				'key' => $this->key,
-				'to' => $this->getToAddresses($message),
-				'raw_message' => (string) $message,
-				'async' => false,
-			],
-		]);
+		$data = [
+			'key' => $this->key,
+			'to' => $this->getToAddresses($message),
+			'raw_message' => (string) $message,
+			'async' => false,
+		];
+
+		if (version_compare('6', $client::VERSION) === 1) {
```

Comment:
PHP also doesn't like that. I can't put the `::` there. You get a parse error. It also doesn't help that I forgot to require guzzle in composer too...